src/platform/test/functional/apps/discover/background_search/_classic.ts

```diff
@@ -47,27 +44,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     describe('Classic view', function () {
-      it('stores a running search', async () => {
```

Comment:
note that we should aim to add a test for it, but it's also  fine to remove it now, since it needs more investment, and e.g. doesn't work in serverless

Reply:
we need to take care of some functionals after enabling background search as a default anyway , I'm currently skipping some, so there will be a follow up issue to fix it. It's mostly that we no longer have the searchSessionIndicator present 

https://github.com/elastic/kibana/pull/236266/commits/5e1cbdf362498678ceef994da58a48c922890916

and changing serverless permissions